bitnami/spark/README.md

```diff
@@ -0,0 +1,266 @@
+| `security.ssl.protocol`                     | Set the SSL protocol                                                                        | `TLSv1.2`                                               |
+| `security.certificatesSecretName`           | Set the name of the secret that contains the certificates                                   | No default                                              |
+| `service.type`                              | Kubernetes Service type                                                                     | `ClusterIP`                                             |
+| `service.port`                              | spark client port                                                                           | `2379`                                                  |
+| `service.nodePort`                          | Port to bind to for NodePort service type (client port)                                     | `nil`                                                   |
+| `service.annotations`                       | Annotations for spark service                                                               | {}                                                      |
+| `service.loadBalancerIP`                    | loadBalancerIP if spark service type is `LoadBalancer`                                      | `nil`                                                   |
+| `ingress.enabled`                           | Enable the use of the ingress controller to access the web UI                               | `true`                                                  |
+| `ingress.hosts`                             | Add hosts to the ingress controller with name and path                                      | `name: spark.local`, `path: /`                          |
+
+
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```console
+$ helm install --name my-release \
+  --set master.webPort=8081 bitnami/spark
+```
+
+The above command sets the spark master web port to `8081`.
+
+Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+
+```console
+$ helm install --name my-release -f values.yaml bitnami/spark
+```
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
+
+### Using custom configuration
+
+To use a custom configuration a ConfigMap should be created with the `spark-env.sh` file inside the ConfigMap. The ConfigMap name must be provided at deployment time using `--set master.configurationConfigMap=configMapName`, to set the configuration on the master, or `--set worker.configurationConfigMap=configMapName`, to set the configuration on the worker. It can be set both at the same time with the same ConfigMap or using two ConfigMaps.
```

Comment:
This section does not render nice. Can you add some blank spaces to separate paragraphs?

Reply:
Not sure about what is bad render but I improved the section.